### ISSUE TITLE
xiaohuai姐姐，您好，我想问关于conv2d的里面的padding, 

### ISSUE THREAD

**Issue by wendongj** [Le-Xiaohuai-speech/DPCRN_DNS3#6]
xiaohuai姐姐，我看1.15里面padding那里选项只有两个“valid”或者“same”，就是在您的main.py函数里面，直接运行main.py会报错，我试过tf.pad方法把您的padding列表加进去，然后把conv2d里面的padding改成“valid”或者“same”可是encode的输出width不是50了，我觉得是我哪里弄错了，可是一时也没有想出来，我鼓起勇气想问您一下，之前看到您的论文时候很感兴趣，把dprnn的方法用在了语音增强上面，而且效果也很好。谢谢您开源了您的代码，对于我这种刚入门的小白来说真的好幸运。

**Comment by Le-Xiaohuai-speech** [comment]
首先我是♂，然后1.15.0的tf.keras.Conv2D 的padding选项可以用4维数组传入和tf.padding 形式一样的参数，这是没问题的（如果有问题可能是keras版本不对，推荐你只安装tensorflow，用tensorflow.keras 就好。Conv1D好像padding只有几个固定的参数）。same就是不考虑两端补到对应长度，如果你要额外加入padding应该是用valid。

**Comment by wendongj** [comment]
> 首先我是♂，然后1.15.0的tf.keras.Conv2D 的padding选项可以用4维数组传入和tf.padding 形式一样的参数，这是没问题的（如果有问题可能是keras版本不对，推荐你只安装tensorflow，用tensorflow.keras 就好。Conv1D好像padding只有几个固定的参数）。same就是不考虑两端补到对应长度，如果你要额外加入padding应该是用valid。

**Comment by wendongj** [comment]
> > 首先我是♂，然后1.15.0的tf.keras.Conv2D 的padding选项可以用4维数组传入和tf.padding 形式一样的参数，这是没问题的（如果有问题可能是keras版本不对，推荐你只安装tensorflow，用tensorflow.keras 就好。Conv1D好像padding只有几个固定的参数）。same就是不考虑两端补到对应长度，如果你要额外加入padding应该是用valid。

谢谢学长，应该是我的keras版本不对，多谢
